SPRINT-STRUCTURE.md

```diff
@@ -1,115 +1,73 @@
-      - Participants required
-      - Outputs (Decisions, actions to conclude meeting)
-    - Definition of ready for stories
-      - Examples of well defined acceptance criteria
-      - Can the story be tested as written
-      - Can it be completed within a sprint
-      - Is it dependent on other stories
-    - [Estimation](sprint-planning/estimation/readme.md)
-      - Resolving estimation conflicts (two people are sizing differently))
+### Day 1
 
-## Assumptions
+- [ ] [Plan the first sprint](sprint-planning/readme.md)
+  - Agree on a sprint goal, and how to measure the sprint progress
+  - Determine team capacity
+  - Assign user stories to the sprint and split user stories into tasks
+  - Set up Work in Progress (WIP) limits
+- [ ] [Decide on test frameworks and discuss test strategies](test-first-development/readme.md)
+  - Discuss the purpose and goals of tests and how to measure test coverage
+  - Agree on how to separate unit tests from integration, load and smoke tests
+- [ ] [Decide on branch naming for feature branches](source-control/feature-branching/readme.md)
+- [ ] [Discuss security needs and verify that secrets are kept out of source control](continuous-deployment/secrets-management/recipes/azure-devops/secrets-per-branch.md)
 
-This workflow makes the following assumptions about the development environment
+### Day 2
 
-- Team is using git for version control.
+- [ ] [Set up Source Control](source-control/readme.md)
+  - Agree on [best practices for commits](https://github.com/microsoft/code-with-engineering-playbook/tree/master/source-control#commit-best-practices)
+- [ ] [Set up basic Continuous Integration with linters and automated tests](continuous-integration/readme.md)
+- [ ] [Set up meetings for Daily Standups and decide on a scrum master](stand-ups/readme.md)
+  - Discuss purpose, goals, participants and facilitation guidance
+  - Discuss timing, and how to run an efficient stand-up
+- [ ] [If the project has sub-teams, set up a Scrum of Scrums](scrum-of-scrums/readme.md)
+
+### Day 3
+
+- [ ] [Agree on code style](code-reviews/README.md) and on [how to assign Pull Requests](code-reviews/pull-requests.md)
+- [ ] [Set up Build Validation for Pull Requests (2 reviewers, linters, automated tests)](code-reviews/README.md) and agree on [Definition of Done](team-agreements/definition-of-done/readme.md)
+- [ ] [Agree on a Code Merging strategy](source-control/git.md) and update the [CONTRIBUTING.md](resources/templates/CONTRIBUTING.md)
+- [ ] [Agree on logging and observability frameworks and strategies](observability/readme.md)
+
+### Day 4
+
+- [ ] [Set up Continuous Deployment](continuous-deployment/readme.md)
+  - Determine what environments are appropriate for this solution
+  - For each environment discuss purpose, when deployment should trigger, pre-deployment approvers, sing-off for promotion.
+- [ ] [Desing a feature and conduct a Design Review](design-reviews/readme.md)
```

Comment:
Should this be rephrased to "Agree on how to [Design a feature and conduct a Design Review](design-reviews/readme.md)". It